python_examples/megno/problem.py

```diff
@@ -32,7 +32,7 @@ def setup_planet(com, mass, period, M, omega, eccentricity):
     if eccentricity>0.8:
         E = math.pi
     F = E - eccentricity*math.sin(M) - M
-    for i in xrange(100):
+    for i in range(100):
```

Comment:
Note that this works in python 2.0 -- but it will use the "bad" `range` function. If you overwrite `range` to be `xrange` in the  module, it will not change the namespace in the `problem.py` file.